apps/preview/next/pages/showcases/Footer/FooterBasic.tsx

```diff
@@ -0,0 +1,267 @@
+            {subcategories?.map(({ subcategoryLabel, link }) => (
+              <SfListItem className="py-2 !bg-transparent typography-text-sm font-body" key={subcategoryLabel}>
+                <SfLink
+                  className="no-underline text-neutral-600 hover:underline hover:!text-neutral-900 active:underline active:!text-neutral-900"
+                  variant="secondary"
+                  href={link}
+                >
+                  {subcategoryLabel}
+                </SfLink>
+              </SfListItem>
+            ))}
+          </div>
+        ))}
+      </div>
+      <hr />
+      <div className="py-10 md:flex md:mx-auto max-w-[1536px]">
+        {contactOptions.map(({ label, icon, link, details }) => {
+          const Icon = iconsMap[icon];
+          return (
+            <div className="mx-auto my-4 text-center" key={label}>
+              <Icon size="lg" />
+              <p className="py-1 my-2 font-medium typography-text-lg font-body">
+                <SfLink
+                  variant="secondary"
+                  className="no-underline text-neutral-600 hover:underline hover:!text-neutral-900 active:underline active:!text-neutral-900"
+                  href={link}
+                >
+                  {label}
+                </SfLink>
+              </p>
+              {details?.map((option) => (
+                <p className="text-sm leading-5 text-neutral-600 font-body" key={option}>
+                  {option}
+                </p>
+              ))}
+            </div>
+          );
+        })}
+      </div>
+      <div className="bg-neutral-900 justify-end px-4 py-10 md:flex md:py-6 max-w-[1536px] mx-auto">
+        <div className="flex justify-center py-2 gap-x-4 md:self-start">
+          {socialMedia.map(({ icon, label, link }) => {
+            const Icon = iconsMap[icon];
+            return (
+              <SfButton
+                key={label}
+                square
+                as="a"
+                variant="tertiary"
+                className="text-white active:text-white hover:text-white hover:!bg-neutral-500 active:bg-transparent"
+                href={link}
+                aria-label={`Go to ${label} page`}
+              >
+                <Icon />
+              </SfButton>
+            );
+          })}
+        </div>
+        <div className="flex items-center justify-center gap-6 py-2 my-4 md:ml-auto md:my-0">
+          {bottomLinks.map(({ label, link }) => (
+            <SfLink
+              key={label}
+              variant="secondary"
+              className="text-white no-underline typography-text-sm active:text-white active:underline hover:text-white hover:underline"
+              href={link}
+            >
+              {label}
+            </SfLink>
+          ))}
+        </div>
+        <p className="flex items-center justify-center py-2 text-sm leading-5 text-center text-white/50 font-body md:ml-6">
```

Comment:
Same `text-sm` class here. Please also check it in Vue block.

Reply:
fixed